package.json

```
@@ -3,21 +3,31 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev": "./node_modules/.bin/parcel ./static/index.html",
-    "build": "./node_modules/.bin/parcel build ./static/index.html"
+    "dev": "parcel ./static/index.html",
+    "build": "parcel build ./static/index.html",
+    "lint": "npx eslint ./ --fix"
```

у тебя eslint установлен локально, но когда ты вызываешь `npx`, то этот пакет в рантайме скачивается и исполняется. Зачем это делать ? предлагаю убрать `npx` и вызывать локально установленный пакет

```suggestion
    "lint": "eslint ./ --fix"
```